tss-esapi/tests/integration_tests/common/mod.rs

```diff
@@ -432,3 +434,45 @@ pub fn create_public_sealed_object() -> Public {
         .build()
         .expect("Failed to create public structure.")
 }
+
+#[allow(dead_code)]
```

Comment:
Is this really necessary?

Reply:
I originally followed the example of most other functions in this file, but apparently it was not.